docs/WritingTests.asciidoc

```diff
@@ -157,6 +157,37 @@ sub run() {
 }
 --------------------------------------------------------------
 
+In addition to Perl, it is possible to write tests in Python. Although the
+boilerplate is slightly different the function names are the same.
+
+[caption="Example: "]
+.Test for the openQA web UI written in Python
```

Comment:
hm, I thought of referencing the existing code by URL instead of providing it directly here but I guess the example is actually nice as is